src/components/views/create/CreateIndex.tsx

```diff
@@ -31,7 +36,16 @@ const CreateIndex = () => {
 		}
```

Comment:
Just like in the Edit view, with this flow I believe we can remove the automatic welcome modal and use this new design, is more telling for the user instead of just forcing a login

Reply:
I talked with @MoeNick and he said opening welcome and sign in modal automatically is in our design flow and we can't change it.